.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+python:
+    - 2.7
+install:
+    - pip install -e .
+    - pip install coveralls
+script:
+    - python setup.py test --coverage --pep8 --flakes
```

Comment:
Just out of curiosity, is there any particular reason you prefer pyflakes to pylint?